src/EprPrnIntegration.Api/settings.json

```diff
@@ -1,23 +1,35 @@
 {
   "IsEncrypted": false,
   "Values": {
+    "FeatureManagement:RunIntegration": true, // feature flag
+    "DefaultLastRunDate": "2024-01-01",
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
     "HOST": "localhost",
     "Functions:Worker:HostEndpoint": "http://localhost:7071",
     "AzureKeyVaultUrl": "https://devrwdinfkv1401.vault.azure.net/",
-    "NpwdApiBaseUrl": "https://fat.npwd.org.uk/",
     "UpdateProducersTrigger": "0 0 18 * * 1-5",
+    "UpdatePrnsTrigger": "0 0 18 * * 1-5",
+    "NpwdApiBaseUrl": "https://fat.npwd.org.uk/",
     "ProducersContext": "https://fat.npwd.org.uk/odata/Producers/$delta",
     "UpdatePrnsTrigger": "0 0 18 * * 1-5",
     "PrnsContext": "https://fat.npwd.org.uk/odata/PRNs/$delta",
+    "Service:Url": "https://devrwdwebwa7407.azurewebsites.net/",
     "Service:EndPointName": "api/organisations",
+    "Service:PrnBaseUrl": "https://devrwdwebwa7418.azurewebsites.net/",
+    "Service:PrnEndPointName": "api/v1/prn",
     "FetchNpwdIssuedPrns:Schedule": "0 0 18 * * 1-5",
-    "ServiceBus:FullyQualifiedNamespace": "UseDevelopmentStorage=true",
-    "ServiceBus:FetchPrnQueueName": "UseDevelopmentStorage=true",
-    "ServiceBus:ErrorPrnQueue": "UseDevelopmentStorage=true",
-    "FeatureManagement:RunIntegration": true, // feature flag
-    "ServiceBus:DeltaSyncQueueName": "UseDevelopmentStorage=true",
-    "DefaultLastRunDate": "2024-01-01"
+    "ServiceBus:FullyQualifiedNamespace": "DEVRWDINFSB1402.servicebus.windows.net",
+    "ServiceBus:FetchPrnQueueName": "defra.epr.npwd.integration.fetchprn.dev7",
+    "ServiceBus:ErrorPrnQueue": "defra.epr.npwd.integration.errorqueue.dev7",
+    "ServiceBus:DeltaSyncQueueName": "defra.npwd.integration.lastsuccessfulrun.dev7",
+    "ServiceBus:MaxWaitTime": "1",
+    "MessagingConfig:ApiKey": "",
+    "MessagingConfig:PrnTemplateId": "a43a573c-3367-4aa2-b9ac-6b347ce8bf32",
+    "MessagingConfig:PERNTemplateId": "24925202-125b-4b85-b54a-9f3100301eb4",
+    "MessagingConfig:NpwdEmailTemplateId": "4e67b91b-b010-4d39-a0ea-ecb7e5e77b25",
+    "MessagingConfig:NpwdEmail": "npwd.support@solution7.co.uk",
```

Comment:
I'm wondering if we add this for dev as well this mail will start receiving mail once we start testing. Probably this mail is for Prod. Can we check with Suketu what can be used for Dev. If we don't have then for testing we can use ours and replace with any dummy value.

Reply:
Had discussion with masil , have informed him to to use dev's email id for dev env